tests/test_transforms.py

```diff
@@ -102,6 +104,22 @@ def return_df(one, dataset_path, branch):
     }.get(dataset_path)
 
 
+def return_path(one, identity, branch):
+    dataset_path = identity["dataset_path"]
+    df = return_df(one, dataset_path, branch)
+    destination = os.sep.join(
+        [
+            foundry_dev_tools.config.Configuration[
+                "cache_dir"
+            ],  # this is a tmp dir in unit tests
+            identity["dataset_rid"],
+            identity["last_transaction_rid"] + ".parquet",
+        ]
+    )
+    df.write.parquet(os.path.join(destination, "spark"))
```

Comment:
@schmelczer No idea why but I had to add `mode='overwrite'` to make the tests go through in my codespace
`df.write.parquet(os.path.join(destination, "spark"), mode='overwrite')`